src/AirshipBindings.iOS.MessageCenter/ApiDefinitions.cs

```diff
@@ -47,14 +46,17 @@ partial interface Constants
         // extern NSString * const UAUserCreatedNotification
         [Field("UAUserCreatedNotification", "__Internal")]
         NSString UAUserCreatedNotification { get; }
-
     }
 
     // @interface UADefaultMessageCenterListViewController : UIViewController <UITableViewDelegate, UITableViewDataSource, UIScrollViewDelegate>
     [BaseType(typeof(UIViewController))]
     interface UADefaultMessageCenterListViewController : IUITableViewDelegate, IUITableViewDataSource, IUIScrollViewDelegate
     {
-        // @property (nonatomic, strong, readwrite) UAMessageCenterStyle *_Nonnull style;
+        // @property (nonatomic, strong, readwrite) UAMessageCenterStyle *_Nonnull messageCenterStyle;
+        [Export("messageCenterStyle", ArgumentSemantic.Strong)]
+        UAMessageCenterStyle MessageCenterStyle { get; set; }
+
+        // @property(nonatomic, strong) UAMessageCenterStyle *style
         [Export("style", ArgumentSemantic.Strong)]
         UAMessageCenterStyle Style { get; set; }
```

Comment:
What happens when this is built for iOS 14? If there is a problem, is there anything we can do?

Reply:
Oh you know that's a good question.

Reply:
I'm going to rebind these to all point to the new properties, which should cover all our bases since they will always be present and will always work.